pkg/kubelet/runtimeclass/runtimeclass_manager.go

```diff
@@ -76,3 +76,8 @@ func (m *Manager) LookupRuntimeHandler(runtimeClassName *string) (string, error)
 
 	return rc.Handler, nil
 }
+
+// Lister returns the runtime class lister.
+func (m *Manager) Lister() nodev1.RuntimeClassLister {
```

Comment:
Let's just expose a `List()` function that lists everything instead.

Reply:
The code is removed